content/millennium/r4/financial.md

```diff
@@ -7,4 +7,5 @@ layout: api
 
 * [Account](../financial/account)
 * [Coverage](../financial/coverage)
```

Comment:
I think ChargeItem should be listed before coverage for alphabetic order.

Reply:
updated here: https://github.com/cerner/fhir.cerner.com/pull/605/commits/64ddf564c76cd6938adb0bdff00feda5cff09c86#diff-0eeb022ddf17088bd7f896481e3b1ee471fe27e9abaf286a5a6b3e22bbfb97a9L9